tasks/release.ts

```diff
@@ -0,0 +1,143 @@
+export = function(grunt: IGrunt, packageJson: any) {
+	const execa = require('execa');
+	const npmPath = 'node_modules/.bin/npm';
+	const temp = 'temp/';
+	const preReleaseTags = ['alpha', 'beta', 'rc'];
+
+	const releaseVersion = grunt.option<string>('release-version');
```

Comment:
you could do:

```typescript
const releaseVersion: string = grunt.option('release-version');
```

It reads a bit better to me, but I am open to alternative viewpoints